docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -130,7 +130,8 @@ docker-compose down
 
 ## Connect to a Solana cluster RPC endpoint
 
-**Command Line Options**
+**RPC endpoints**
+
 A Proxy connects to a public [Solana cluster RPC endpoint](https://docs.solana.com/cluster/rpc-endpoints) depending on the `SOLANA_URL` value set. The following table shows the *endpoint* value that's set automatically based on the value of the `CONFIG` flag.
 
 To use a different endpoint, you need to specify the variable `-e SOLANA_URL='http://<Solana node RPC endpoint>'` on the command line. For example, to use Devnet, add the flag `-e SOLANA_URL='https://api.devnet.solana.com'`.
```

Comment:
Please let's remove this line where we again advise users to setup `SOLANA_URL` as one of the public Solana nodes. Here it's https://api.devnet.solana.com

Reply:
Of course. Thanks!